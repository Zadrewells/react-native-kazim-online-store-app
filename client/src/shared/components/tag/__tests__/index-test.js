import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import {
	Tag
} from 'src/shared/components';

describe('<Tag />', () => {
	const getWrapper = props => renderer.create(
		<Tag
			{
			...props
			}
		/>
	).toJSON();

	const modelI = {
		width: 50,
		height: 50,
		primary: true
	};

	it('renders correctly', () => {
		const tree = getWrapper(modelI);
		expect(tree).toMatchSnapshot();
	});
});
