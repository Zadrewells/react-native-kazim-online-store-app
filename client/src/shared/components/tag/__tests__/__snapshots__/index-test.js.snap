// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tag /> renders correctly 1`] = `
<View
  height={50}
  primary={true}
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#5197ff",
        "borderRadius": 50,
        "display": "flex",
        "height": 50,
        "justifyContent": "center",
        "width": 50,
      },
    ]
  }
  width={50}
/>
`;
